### PR TITLE
Chore/upgrade to richie beta 22

### PR DESCRIPTION
**Pull request by jbpenrath**
### Fixed

- Add missing factory-boy dependency to allow generating the demo site

### Changed

- Upgrade richie to 2.0.0-beta.22

### Changed

- Upgrade richie to 2.0.0-beta.22

### Changed

- Upgrade richie to 2.0.0-beta.22

### Removed

- Remove gimporter app

### Changed

- Upgrade richie to 2.0.0-beta.22

### Changed

- Upgrade richie to 2.0.0-beta.22

### CODE REVIEW

File: sites/demo/requirements/base.txt [hunk 1]
```diff
@@ -4,5 +4,5 @@ django-storages==1.10.1
 dockerflow==2020.10.0
 gunicorn==20.0.4
 psycopg2-binary==2.8.6
-richie==2.0.0-beta.21
+richie==2.0.0-beta.22
 sentry-sdk==0.19.1
```
Comment (sampaccoud) [inline sites/demo/requirements/base.txt:8]: can you add factory as a base dependency?

File: sites/funmooc/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.17.0] - 2020-12-07
+### Changed
```
Comment (sampaccoud) [inline sites/funmooc/CHANGELOG.md:12]:
```suggestion
## [0.17.0] - 2020-12-07

### Changed
```

File: sites/funmooc/src/backend/funmooc/settings.py [hunk 3]
```diff
@@ -309,7 +309,6 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     INSTALLED_APPS = (
         # Funmooc stuff
         "base",
-        "gimporter",
```
Comment (sampaccoud) [inline sites/funmooc/src/backend/funmooc/settings.py:312]: you should also remove from requirements/base.txt:
```
# Google sheet importer
gspread==3.1.0
markdown==3.1.1
oauth2client==4.1.3
```

File: sites/demo/requirements/base.txt (hunk 1 above)
Comment (jbpenrath) [reply sites/demo/requirements/base.txt:8]: Ah ah 😅
https://github.com/openfun/richie-site-factory/commit/d72e014bb8e38de3dbf5b7ac4b0d6d1a7f0cbc4d

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**